src/services/overlay-fallback/overlay-fallback.service.ts

```diff
@@ -0,0 +1,149 @@
+import { ConnectedPosition, FlexibleConnectedPositionStrategy, HorizontalConnectionPos, OriginConnectionPosition, OverlayConnectionPosition, OverlayRef, VerticalConnectionPos } from '@angular/cdk/overlay';
+import { Injectable } from '@angular/core';
+import { AnchorPlacement } from '../../components/tooltip';
```

Comment:
You do have at least one circular dependency here, the fix for this one would be to move the AnchorPlacement type definition to a separate file alongside this service. See if that helps.

Reply:
Thanks for this, however it didn't fix the issue I am facing